setup.py

```diff
@@ -1,15 +1,35 @@
-from cx_Freeze import setup, Executable
+from setuptools import setup, find_packages
+from io import open
+import qtsass
 
-build_exe_options = {
-    'include_msvcr': True,
-    'bin_includes': ['msvcp120.dll'],  # Force msvcp120 to be included (libsass dependency, not copied by include_mscvr)
-    'optimize': 2
-}
 
 setup(
-    name="qtsass",
-    description="Compile a SCSS file to a valid Qt CSS.",
-    executables=[Executable("qtsass/qtsass.py")],
-    options={'build_exe': build_exe_options},
-    requires=['libsass', 'cx_Freeze', 'watchdog'],
+    name=qtsass.__title__,
```

Comment:
This could be potentially problematic (having to import the package itself)

So I would prefer we just left the version on `__init__.py` and the rest here